### DISCUSSION TITLE
Dusk to Dawn

### DISCUSSION THREAD

**Discussion by davlaw** [themactep/thingino-firmware#379] [Q&A]
Sorry for noob question, just brought Thingino into my system this week.

What does Dusk to Dawn script do and how if any interference there might be with Day/Night Mode Control?  Guess curious about proper config for this feature?

**Comment by tERB00** [comment]
Busk 2 Dawn is a timed day/night switch based on your geographical location regardless of real illumination conditions.

**Reply by davlaw** [reply]
So would I understand that to override the d/n control that is based on gain?

**Reply by tERB00** [reply]
Yes, the default day/night script is being triggered by max gain in day mode and min gain in night mode.

**Reply by davlaw** [reply]
But the settings when "daynight" is run with "day" or  "night" arguments is set based on settings set in control option.  Got it.  Finally ssh into cam , ran with changes saw results.  Easy enough, question if these 2 settings should be in Day/Night Mode Control. The way it was presented made me thing they were 2 totally different options etc.  Sorry for noob question, just have a fews days under my belt.

Thanks
